man/tpm2_evictcontrol.1.md

```diff
@@ -18,9 +18,8 @@ be evicted.
 # OPTIONS
 
   * **-A**, **--auth**=_AUTH_:
-    The authorization used to authorize the commands. Valid choices are:
-    *  **o** for **TPM_RH_OWNER**
-    *  **p** for **TPM_RH_PLATFORM**
+    The authorization used to authorize the commands.
+    [hierarchy specifiers](common/hierarchy.md)
```

Comment:
For storage, there are only platform and owner hierarchies, I don't think you can use the endorsement and null hierarchies so this wouldn't be correct.

Reply:
The same comment applies to all commands that work with the NVRAM.

Reply:
Yeah I was wondering about how we wanted to handle that. We make the common parsing routine take some flags for valid options, but at the same time, anything invalid will be rejected by the TPM.